march_rqt_robot_monitor/src/march_rqt_robot_monitor/updater.py

```diff
@@ -0,0 +1,23 @@
+
+import diagnostic_updater
+import rospy
+from std_msgs.msg import Time
+
+from .diagnostic_analyzers.temperature import temperature_updater
+from .diagnostic_analyzers.topic_frequency import CheckTopicFrequency
+
+
+def main():
+    rospy.init_node('Diagnostic_updater')
+
+    updater = diagnostic_updater.Updater()
+    updater.setHardwareID('MARCH IV')
```

Comment:
;)
```suggestion
    updater.setHardwareID('MARCH IVc')
```

Reply:
You might want to change this to a parameter

Reply:
nice.